lib/components/user_input/Select.tsx

```diff
@@ -47,12 +61,18 @@ export const Select = <T, >({ value, label, options, onChange, isHidingCurrentVa
                 <Menu.Item key={option.label}>
                   {
                     <div
-                      className={tx('px-4 py-2 cursor-pointer overflow-hidden whitespace-nowrap text-ellipsis hover:bg-gray-100 border-2 border-t-0', {
-                        'bg-gray-100 ': option.value === value,
+                      className={tx('px-4 py-2 overflow-hidden whitespace-nowrap text-ellipsis border-2 border-t-0', {
+                        'bg-gray-100': option.value === value,
                         'bg-gray-50': index % 2 === 1,
+                        'text-gray-300 cursor-not-allowed': option.disabled,
```

Comment:
```suggestion
                        'text-gray-300 cursor-not-allowed': !!option.disabled,
```
Either we require `disabled: boolean` (non null) or casting this value to a boolean.